kover-cli/src/main/kotlin/kotlinx/kover/cli/commands/ReportCommand.kt

```diff
@@ -47,8 +47,8 @@ internal class ReportCommand : Command {
     @Option(name = "--html", usage = "generate a HTML report in the specified path", metaVar = "<html-dir>")
     private var htmlDir: File? = null
 
-    @Option(name = "--title", usage = "title in the HTML report", metaVar = "<html-title>")
-    private var htmlTitle: String? = null
+    @Option(name = "--title", usage = "title in the HTML or XML report", metaVar = "<title>")
+    private var title: String? = null
```

Comment:
Is this change related to the fix in any way?

Reply:
Without these changes, a compilation error occurs :)